test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -130,6 +131,27 @@ public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
         pipelineWatcher = new PipelineWatcher(containerComposer);
     }
     
```

Comment:
1, `createProxyDatabase` should be invoked after `cleanUpPipelineJobs`, since `DROP DATABASE` might check uncompleted jobs.

2, `createProxyDatabase` should be invoked in every unit test method.